packs/equipment-effects/effect-ichthyosis-mutagen.json

```diff
@@ -24,6 +24,16 @@
                 "key": "FlatModifier",
                 "selector": "reflex",
                 "value": -2
+            },
+            {
+                "itemType": "condition",
+                "key": "ItemAlteration",
+                "mode": "override",
+                "predicate": [
+                    "item:damage:type:bleed"
+                ],
+                "property": "pd-recovery-dc",
+                "value": 1
```

Comment:
Does this auto succeed on a nat 1? The system will still downgrade the success to a failure there I think.,

Reply:
This would be done better with a DoS adjustment

Reply:
I agree

Reply:
That's a good idea!

It looks like the damage type isn't being passed to the "flat-check" selector so I've reverted the changes to this file, and I'll submit a separate PR.